_overviews/toolkit/munit-assertions.md

```diff
@@ -1,18 +1,218 @@
 ---
 title: How to write assertions?
 type: section
-description: ???
+description: Comparing what you obtain with what you expect
 num: 4
 previous-page: munit-test-suite
 next-page: munit-exceptions
 ---
 
 {% include markdown.html path="_markdown/install-munit.md" %}
 
-<!--
-Tutorial about `assert`, `assertEquals`, `assertNotEquals` and `assertNoDiff`
+Assertions describe what to check in your tests. If any assertion in a unit test
+fails, then the test fails.
 
-https://scalameta.org/munit/docs/assertions.html
+Assuming you have defined a
+[test suite]({% link _overviews/toolkit/munit-test-suite.md %}) `MyTests`, this
+tutorial shows various ways to write assertions within test specifications.
 
-We explain clues in a later tutorial.
- -->
\ No newline at end of file
+{% tabs assertions-0 class=tabs-scala-version %}
+{% tab 'Scala 2' %}
+```scala
+class MyTests extends munit.FunSuite {
+  test("a unit test") {
+    // test specification goes here
+  }
+}
+```
+{% endtab %}
+{% tab 'Scala 3' %}
+```scala
+class MyTests extends munit.FunSuite:
+  test("a unit test") {
+    // test specification goes here
+  }
+```
+{% endtab %}
+{% endtabs %}
+
+## Comparing What You Get With What You Expect
```

Comment:
Most of the people don't read everything, and we should make it easy for them to find the information they are looking for.

So here the title could simply be ```## assertEquals```, as it is in the [MUnit document](https://scalameta.org/munit/docs/assertions.html#assert).

Reply:
Do you mean that people coming here already know the solution they are looking for? I think it is better to use a title that relates to the problem they want to solve rather than the solution, no?

Reply:
Yes I think we can assume that people know what an assertion is. Because this is not a course to teach programming, but a tutorial on how to write assertions in Scala.

The typical user would be someone knowing already about programming, that just don't know or don't remember how to write assertions, because they don't use Scala very oftenn or because they use MUnit for the first time. By writing `assertEquals` in big we make it easy for them to find the information they need.

Reply:
> The typical user would be someone knowing already about programming, that just don't know or don't remember how to write assertions, because they don't use Scala very often or because they use MUnit for the first time. By writing `assertEquals` in big we make it easy for them to find the information they need.

I wonder if that audience would directly go to the MUnit documentation instead of coming here.

Reply:
That's a good point.